sympy/printing/tests/test_pycode.py

```diff
@@ -70,6 +79,29 @@ def test_pycode_reserved_words():
     assert py_str in ('else_ + if_', 'if_ + else_')
 
 
+def test_sqrt():
+    prntr = PythonCodePrinter()
+    assert prntr._print_Pow(sqrt(x), rational=False) == 'math.sqrt(x)'
+    assert prntr._print_Pow(sqrt(x), rational=True) == 'x**(1/2)'
+
+    prntr = MpmathPrinter()
+    assert prntr._print_Pow(sqrt(x), rational=False) == 'mpmath.sqrt(x)'
+    assert prntr._print_Pow(sqrt(x), rational=True) == \
+        'x**(mpmath.mpf(1)/mpmath.mpf(2))'
```

Comment:
```suggestion
        'x**(mpmath.mpf(0.5))'
```

Reply:
This is optional...there's little difference in the two.

Reply:
I think that different precedence control should be implemented, to prevent it from adding additional brackets.